falcon/__init__.py

```diff
@@ -25,9 +25,9 @@
 
 import logging as _logging
 
-# Hoist classes and functions into the falcon namespace
-# Explicitly list all export here, unless there are many export of the
-# same type (example status codes, constans and errors)
+# Hoist classes and functions into the falcon namespace. Explicitly list
+#   all export here, unless there are many export of the
+#   same type (example status codes, constans and errors)
```

Comment:
Old stuff, but while at it, probably it would sound more natural as plural _all exports_ here, also _constans_ (sic) :arrow_right: _constants_.

Reply:
updated